browsers/firefox/manifest.json

```diff
@@ -55,7 +55,8 @@
         },
         {
             "js": [
-                "public/js/content-scripts/autofill.js"
+                "public/js/content-scripts/autofill.js",
+                "public/js/content-scripts/breakage-stats.js"
```

Comment:
This one only needs to run in the main frame, and probably don't need to be run on `document_start`, but can run later. Let's create a new content-script entry for it.

Reply:
I created a new entry to only target main frames but found it was still necessary to run at document_start. Otherwise page reloads weren't captured.